internal/cloudinit/cloudinit.go

```diff
@@ -249,6 +249,17 @@ echo "done starting lite engine server"
 groupadd docker
 mkdir -p /opt/gitspaceagent
 
+echo "Updating docker root dir"
+systemctl stop docker
+mkdir -p /mnt/disks/mountdevcontainer/docker
```

Comment:
If we add the docker folder like this, we need to update the path which will be used by the gitspace as the source for bind mount. We do not want the gitspace to access the docker data dir.
Do we have the capability to return the storage path from OVH VM?

Reply:
Creating /mnt/disks/mountdevcontainer/gitspace below. 
Gitspace agent has been updated to point to this dir